app/src/main/java/org/jellyfin/mobile/MainActivity.kt

```diff
@@ -213,4 +215,11 @@ class MainActivity : AppCompatActivity(), WebViewController {
         webView.destroy()
         super.onDestroy()
     }
+
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+        super.onActivityResult(requestCode, resultCode, data)
+        if(requestCode == Constants.HANDLE_EXTERNAL_PLAYER) {
+            ExternalPlayer.handleActivityResult(this, resultCode, data)
+        }
+    }
```

Comment:
You could move that above the `onRequestPermissionResult`, I think that'd be a bit neater.